Makefile

```diff
@@ -15,7 +15,7 @@ help: #! Show this help message
 
 .PHONY: install
 install:
-	@poetry install
+	@cd packages/sdk/server-ai && poetry install
```

Comment:
I'm not sure it makes sense to have the make file for the server-ai at the root if we are moving to a mono repo.